crawlclima/redemet/rmet.py

```diff
@@ -46,7 +50,7 @@ def clean_line(line):
 
 
 def parse_page(page):
-    lines = filter(clean_line, page.split("\n"))
+    lines = filter(clean_line, page.split("\n"))  # < verify this
```

Comment:
>   # < verify this
if you want to verify that in a follow up PR, open an issue for that

Reply:
This comment was only to debug parse with pdb. Removed.